### ISSUE TITLE
[BUG]would it be possible to have Python-3.12 wheels ?

### ISSUE THREAD

**Issue by stonebig** [pydata/bottleneck#439]
**Describe the bug**
Whish of Windows Python-3.12 wheel

**Comment by RNarayan73** [comment]
Following error occurs on installing in Python 3.12 environment:
```
(skl312) PS C:\Users\naray\Python\venvs\skl312> pip install bottleneck

Collecting bottleneck
  Downloading Bottleneck-1.3.7.tar.gz (103 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 103.1/103.1 kB 3.0 MB/s eta 0:00:00
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Requirement already satisfied: numpy in c:\users\naray\python\venvs\skl312\lib\site-packages (from bottleneck) (1.26.3)
Building wheels for collected packages: bottleneck
  Building wheel for bottleneck (pyproject.toml) ... error
  error: subprocess-exited-with-error

  × Building wheel for bottleneck (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [57 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-312
      creating build\lib.win-amd64-cpython-312\bottleneck
      copying bottleneck\_pytesttester.py -> build\lib.win-amd64-cpython-312\bottleneck
      copying bottleneck\_version.py -> build\lib.win-amd64-cpython-312\bottleneck
      copying bottleneck\__init__.py -> build\lib.win-amd64-cpython-312\bottleneck
      creating build\lib.win-amd64-cpython-312\bottleneck\benchmark
      copying bottleneck\benchmark\autotimeit.py -> build\lib.win-amd64-cpython-312\bottleneck\benchmark
      copying bottleneck\benchmark\bench.py -> build\lib.win-amd64-cpython-312\bottleneck\benchmark
      copying bottleneck\benchmark\bench_detailed.py -> build\lib.win-amd64-cpython-312\bottleneck\benchmark
      copying bottleneck\benchmark\__init__.py -> build\lib.win-amd64-cpython-312\bottleneck\benchmark
      creating build\lib.win-amd64-cpython-312\bottleneck\slow
      copying bottleneck\slow\move.py -> build\lib.win-amd64-cpython-312\bottleneck\slow
      copying bottleneck\slow\nonreduce.py -> build\lib.win-amd64-cpython-312\bottleneck\slow
      copying bottleneck\slow\nonreduce_axis.py -> build\lib.win-amd64-cpython-312\bottleneck\slow
      copying bottleneck\slow\reduce.py -> build\lib.win-amd64-cpython-312\bottleneck\slow
      copying bottleneck\slow\__init__.py -> build\lib.win-amd64-cpython-312\bottleneck\slow
      creating build\lib.win-amd64-cpython-312\bottleneck\src
      copying bottleneck\src\bn_config.py -> build\lib.win-amd64-cpython-312\bottleneck\src
      copying bottleneck\src\bn_template.py -> build\lib.win-amd64-cpython-312\bottleneck\src
      copying bottleneck\src\__init__.py -> build\lib.win-amd64-cpython-312\bottleneck\src
      creating build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\common.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\input_modification_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\list_input_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\memory_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\move_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\nonreduce_axis_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\nonreduce_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\reduce_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\scalar_input_test.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\test_template.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\util.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      copying bottleneck\tests\__init__.py -> build\lib.win-amd64-cpython-312\bottleneck\tests
      creating build\lib.win-amd64-cpython-312\bottleneck\tests\data
      creating build\lib.win-amd64-cpython-312\bottleneck\tests\data\template_test
      copying bottleneck\tests\data\template_test\test_template.c -> build\lib.win-amd64-cpython-312\bottleneck\tests\data\template_test
      copying bottleneck\tests\data\template_test\truth.c -> build\lib.win-amd64-cpython-312\bottleneck\tests\data\template_test
      UPDATING build\lib.win-amd64-cpython-312\bottleneck/_version.py
      set build\lib.win-amd64-cpython-312\bottleneck/_version.py to '1.3.7'
      running build_ext
      running config
      compiling '_configtest.c':



      int __attribute__((optimize("O3"))) have_attribute_optimize_opt_3(void*);

      int main(void)
      {
          return 0;
      }

      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for bottleneck
Failed to build bottleneck
ERROR: Could not build wheels for bottleneck, which is required to install pyproject.toml-based projects
```
Can you update the wheels for Python 3.12?

Thanks
Narayan

**Comment by rdbisme** [comment]
Fixed in #442. Tagged, wheel should be available soon.
